SpatialGDK/Source/SpatialGDK/Private/EngineClasses/Components/ActorInterestQueryComponent.cpp

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "EngineClasses/Components/ActorInterestQueryComponent.h"
+
+#include "Schema/Interest.h"
+#include "Interop/SpatialClassInfoManager.h"
+
+UActorInterestQueryComponent::UActorInterestQueryComponent()
+{
+	PrimaryComponentTick.bCanEverTick = false;
```

Comment:
This is set to false in UActorComponent's constructor so not sure we need the constructor at all?

Reply:
👍